legacy-migration/migrate.py

```diff
@@ -99,6 +119,21 @@ def migrate_data_quality_summary():
     end = time.time()
     print(f"Total DataQualitySummary data migration time spent: {end - start} seconds, Succeeded: {success_count}, Failed: {failure_count}")
 
+def migrate_option_definition():
+    global success_count, failure_count
+    success_count= 0
+    failure_count = 0
+    start = time.time()
+    print("Starting OptionDefinition data migration...")
+
+    with oracledb.connect(user=user, password=pwd, dsn="localhost:1521/orcl") as connection:
+        with connection.cursor() as cursor:
+            sql = """select GUID, NAME, DESCRIPTION, FORM, SCOPE, SORT_KEY, DEPRECATED, PROVIDER_GUID from I_10_0_TESTBED_BUSINESS.EJB_OPTION_DEF"""
+            for r in cursor.execute(sql):
+                migrate_od_row(r[0], r[1], r[2], r[3], r[4], r[5], r[6], r[7])
+
+    end = time.time()
+    print(f"Total OptionDefinition data migration time spent: {end - start} seconds, Succeeded: {success_count}, Failed: {failure_count}")
```

Comment:
print(f"Total - is this correct?

Reply:
The f is for a formatted string to include the {end - start} time as a calculated value, and the {success_count} and {failure_count}